api/models/vpp/interfaces/interface.proto

```diff
@@ -40,27 +40,30 @@ message Interface {
     }
     Unnumbered unnumbered = 9;
 
-    message RxModeSettings {
+    message RxMode {
         // from vpp/build-root/install-vpp-native/vpp/include/vnet/interface.h
-        enum RxModeType {
+        enum Type {
             UNKNOWN = 0;
             POLLING = 1;
             INTERRUPT = 2;
             ADAPTIVE = 3;
             DEFAULT = 4;
         };
-        RxModeType rx_mode = 1;
-        uint32 queue_id  = 2;
-        uint32 queue_id_valid  = 3;
+        uint32 queue = 1;
+        Type mode = 2;
+        bool default_mode = 3; // if enabled, <queue> will be ignored and the given
+                               // <mode> will be used for all queues without explicitly
+                               // selected Rx mode
     }
-    RxModeSettings rx_mode_settings = 10;
+    repeated RxMode rx_mode = 10;
```

Comment:
This would probably be better named as `rx_modes` to better represent that it's array.